File: frontend/src/types/frase.ts
export interface Frase {
  id: string;
  categoria: string | null;
  subcategoria: string | null;
  tipo: string | null;
  texto_es: string | null;
  texto_ay: string | null;
  texto_qu: string | null;
  seccion: string | null;
  requiere_revision: boolean | null;
  candidato_audio: boolean | null;
  estado: string | null;
}

File: backend/app/models/frase.py
from sqlalchemy import Column, String, Text, Boolean
from app.core.database import Base

class Frase(Base):
    __tablename__ = "frases"

    id = Column(String, primary_key=True, index=True)
    categoria = Column(String)
    subcategoria = Column(String)
    tipo = Column(String)
    texto_es = Column(Text)
    texto_ay = Column(Text)
    texto_qu = Column(Text)
    seccion = Column(String)
    requiere_revision = Column(Boolean)
    candidato_audio = Column(Boolean)
    estado = Column(String)
